hugo/content/blog/2021-12-07-new-versioning-scheme/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: New versioning scheme
+author: Robert Adam
+date: 2021-12-07
+categories:
+  - News
+---
+Mumble traditionally used a versioning scheme of the form _major.minor.patch_ where the patch version number was incremented for every bug-fix release
+of a stable release series (e.g. 1.3.0 -> 1.3.1). Most notably though, the patch version was always starting at zero for a new stable release series
+(e.g. 1.2.0, 1.3.0, etc.). This will no longer be the case.
+
+<!--more-->
+
+## Background
+
+Mumble was always using the [semantic versioning scheme](https://semver.org/) which generally makes it really easy to know what to expect when

```

Comment:
```suggestion
Mumble has always been using the [semantic versioning scheme](https://semver.org/) which generally makes it really easy to know what to expect when

```

Reply:
@Krzmbrzl What do you think about my suggestion? "was" is ambiguous, as it may imply that we're not using semantic versioning anymore (which is not the case).

Reply:
sounds good to me, but the blog post has been released already. Feel free to make the change directly on master though :+1: